Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -274,6 +276,93 @@
 )
 
 
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    def __init__(self, url: str, verify: bool = True):
+        super().__init__(base_url=url, verify=verify)
+
+    def get_url_enrichment(self, headers: dict, params: dict, ok_codes: tuple) -> Response:
```

Comment:
Basically, we don't have to have these methods. we can call the HTTP request (add the relevant parameters to the method).